dotcom-rendering/src/server/prod-server.ts

```diff
@@ -51,9 +51,11 @@ export const prodServer = (): void => {
 	app.use(requestLoggerMiddleware);
 	app.use(compression());
 
-	app.get('/_healthcheck', (req: Request, res: Response) => {
+	const healthcheck: Handler = (req: Request, res: Response) => {
```

Comment:
The beauty of this is that you no longer have to be explicit about the parameters:

```suggestion
	const healthcheck: Handler = (req, res) => {
```